app/models/user.rb

```diff
@@ -49,6 +49,16 @@ class User < ActiveRecord::Base
   has_many :notes, -> { order 'created_at DESC' }
   has_many :deeds
 
+  has_many :random_collections,   -> { unrestricted.sample },
+    class_name: "Collection",  :foreign_key => "owner_user_id"
+  has_many :random_document_sets, -> { unrestricted.sample }, 
+    class_name: "DocumentSet", :foreign_key => "owner_user_id"
+
+  scope :owners,         -> { where(owner: true) }
+  scope :trial_owners,   -> { owners.where(account_type: 'Trial') }
+  scope :paid_owners,    -> { owners.where('paid_date > ?', Time.now) }
```

Comment:
We can't scope projects by this.

Reply:
I've added `non_trial_owner` to better match the older functionality